persistence/src/test/java/edu/unc/lib/dl/persist/services/transfer/StreamToFSTransferClientTest.java

```diff
@@ -147,4 +175,9 @@ protected void assertContent(Path path, String content) throws Exception {
     protected InputStream toStream(String content) {
         return new ByteArrayInputStream(content.getBytes());
     }
+
+    private Path createFile(Path filePath, String content) throws Exception {
```

Comment:
since you're adding this helper method, you should probably update the existing `ExistingFile` tests to use it since they are all doing it themselves with stuff like `FileUtils.copyInputStreamToFile(toStream(ORIGINAL_CONTENT), binDestPath.toFile());`